README.md

```diff
@@ -151,6 +151,33 @@ This will create a clean environment where is uses the [pip-tools](https://githu
 $ make test
 ```
 
+## Production
+
+In order to get the container ready for production use we need to first build it:
+
+```bash
+$ docker build -t seismic-risc:latest .
+```
+
+Use the `prod.env.dist` template file and create a `prod.env` file with the correct environment variables and run like so:
+
+```bash
+$ docker run --env-file prod.env code4romania/seismic-risc "./start.sh"
```

Comment:
If we're running this from the root directory, it would seem `"./start.sh"` isn't correct, since it's in `seismic_site/`
Am I right?

Reply:
Also, is it normal to get the following message when running this command?
`Unable to find image 'code4romania/seismic-risc:latest' locally`
After I already ran the previous `docker build ...`

Reply:
> If we're running this from the root directory, it would seem `"./start.sh"` isn't correct, since it's in `seismic_site/`
> Am I right?

The `WORKDIR` is already set in the docker image to be inside `/code/seismic_site/` - https://github.com/code4romania/seismic-risc/blob/master/Dockerfile#L25

> Also, is it normal to get the following message when running this command?
> `Unable to find image 'code4romania/seismic-risc:latest' locally`
> After I already ran the previous `docker build ...`

This is a typo. It would work if you already tagged the image to point to the `code4romania` user in docker hub.

> Is it OK for the previous package to be named just `seismic-risc:latest`, without `code4romania/`? Or should the latter command have the `c4ro/` path in it? Or is it just normal for this to happen?

The `code4romania` in the path is for the user in docker hub https://hub.docker.com/u/code4romania under which the image is hosted. Should probably be mentioned only at the last step when we upload the image.

These instructions for running the containers in prod are meant to be guidelines more than ready to use command in your terminal. I think the person in charge of the deployment and devops for the project should update this readme once the process is finalized.

Reply:
Alrighty, then! I don't think there's anything else, then

Reply:
Updated the PR with a clearer README